client/core/core.go

```diff
@@ -1689,10 +1712,28 @@ func (c *Core) assetMap() map[uint32]*SupportedAsset {
 			wallet = w.state()
 		}
 		assets[assetID] = &SupportedAsset{
-			ID:     assetID,
-			Symbol: asset.Symbol,
-			Wallet: wallet,
-			Info:   asset.Info,
+			ID:       assetID,
+			Symbol:   asset.Symbol,
+			Wallet:   wallet,
+			Info:     asset.Info,
+			Name:     asset.Info.Name,
+			UnitInfo: asset.Info.UnitInfo,
+		}
+		for tokenID, token := range asset.Tokens {
+			wallet = nil
+			w, found := c.wallets[tokenID]
+			if found {
+				wallet = w.state()
+			}
+			assets[tokenID] = &SupportedAsset{
+				ID:                    tokenID,
+				Symbol:                dex.BipIDSymbol(tokenID),
+				Wallet:                wallet,
+				Token:                 token,
+				Name:                  token.Name,
+				UnitInfo:              token.UnitInfo,
+				WalletCreationPending: tokenID == creatingWalletID,
```

Comment:
If two token wallets are being created concurrently the one that started first would have this set to false.

Reply:
Was resolved in [9ebb3da](https://github.com/decred/dcrdex/pull/1622/commits/9ebb3dad585d0aa7f39a4961b44a5a92e72604c1), I think.